bigtable/deletes/delete_streaming_batching_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deletes
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/bigtable"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/google/uuid"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestStreamingAndBatching(t *testing.T) {
+	ctx := context.Background()
+	project := os.Getenv("GOLANG_SAMPLES_BIGTABLE_PROJECT")
+	instance := os.Getenv("GOLANG_SAMPLES_BIGTABLE_INSTANCE")
+	if project == "" || instance == "" {
```

Comment:
praise: I love that you're explicitly checking these values up front.

suggestion: call `t.Fatalf()` rather than `t.Skip()`. If these env vars aren't set in the CI/CD, we want to fail and alert the maintainers.